src/Target.php

```diff
@@ -259,23 +270,25 @@ public function getMessagePrefix(array $message): string
      * }
      * ```
      */
-    public function setEnabled($value): void
+    public function setEnabled($value): self
     {
-        $this->_enabled = $value;
+        $this->enabled = $value;
+
+        return $this;
     }
 
     /**
      * Check whether the log target is enabled.
-     * @property bool Indicates whether this log target is enabled. Defaults to true.
      * @return bool A value indicating whether this log target is enabled.
+     * @property bool Indicates whether this log target is enabled. Defaults to true.
```

Comment:
Is this annotation still necessary? Magic getters/setters will no longer work after this change, so there is no virtual properties to document.

Reply:
Not really. Could be removed.

Reply:
Removed